slowmovie.py

```diff
@@ -210,6 +211,17 @@ def estimate_runtime(delay, increment, frames, output="guess"):
 consoleHandler.setFormatter(logging.Formatter("%(message)s"))
 logger.addHandler(consoleHandler)
 
+# check the epd driver - do this first since we can't do much if it fails
+validEpds = displayfactory.list_supported_displays()
+
+if(args.epd not in validEpds):
+    # can't find the driver
+    logging.error(f"{args.epd} is not a valid EPD name, valid names are:")
```

Comment:
```suggestion
    logging.error(f"'{args.epd}' is not a valid EPD name, valid names are:")
```

I think we've been doing that type of quotes with many of the debugging strings